sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1458,7 +1490,7 @@ def task_worker(self):
                         # TODO: Make sure this doesn't impact other datapaths
                         api.set_lpmode(False)
                         self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_AP_CONF
-                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.CMIS_DEF_EXPIRED)
+                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.get_cmis_dp_deinit_duration_secs(api))
```

Comment:
can you log the deinit duration?

Reply:
Sure. I just update the code and the summary of this PR.